src/dbg/_global.h

```diff
@@ -2,7 +2,9 @@
 #define _GLOBAL_H
 
 #define _WIN32_WINNT 0x0501
+#ifndef WINVER

```

Comment:
Why was this addded? The WINVER is defined as `0x0501` because x64dbg should be compiled with Windows XP headers.